exercises/concept/annalyns-infiltration/annalyns-infiltration.js

```diff
@@ -27,7 +27,7 @@
  * @return {boolean} Whether or not you can execute a fast attack.
  */
 export function canExecuteFastAttack(knightIsAwake) {
-  throw new Error('Implement the canExecuteFastAttack function');
+  throw new Error('Remove this line and implement the function');
```

Comment:
Had to remove function names because prettier would make it multiline:
```js
  throw new Error(
    'Remove this line and implement the canExecuteFastAttack function'
  );
```
So `remove this line` might make some students think of just removing the inner string.